RadixWallet/Features/DappInteractionFeature/Children/AccountPermission/AccountPermission+View.swift

```diff
@@ -59,13 +59,12 @@ extension AccountPermission {
 							}
 							.padding(.horizontal, .medium2)
 
-							Spacer()
-
 							Text(L10n.DAppRequest.AccountPermission.updateInSettingsExplanation)
 								.foregroundColor(.app.gray2)
 								.textStyle(.body1Regular)
 								.multilineTextAlignment(.center)
 								.padding(.horizontal, .medium2)
+								.padding(.top, .large1)
```

Comment:
Zeplin indicates the distance between the two elements should be 76, would be `.large1` + `36` (which we don't have any `unit * 9`). Still, this distance seems quite arbitrary, so I would confirm with Aftab that is a fixed distance and not that we want it centered (as it currently shows).

Reply:
`.large1` from VStack spacing + this `.large1` will be 80. I recall Aftab agreed that we should round to the nearest multiple of 4.
Regarding fixed distance vs centered, I will confirm with Aftab and the Android team.

Reply:
but 76 is also multiple of 4, that could be the expected distance as well

Reply:
Discussed with Aftab and he adjusted the spacing to `.medium1`